modules/apps/27-interchain-accounts/controller/keeper/migrations_test.go

```diff
@@ -86,7 +86,10 @@ func (suite *KeeperTestSuite) TestMigratorMigrateParams() {
 			func() {
 				params := icacontrollertypes.DefaultParams()
 				subspace := suite.chainA.GetSimApp().GetSubspace(icacontrollertypes.SubModuleName) // get subspace
-				subspace.SetParamSet(suite.chainA.GetContext(), &params)                           // set params
+				if !subspace.HasKeyTable() {
```

Comment:
will it ever be the case that this is now initialized? I.e can't the `if` be removed completely for testing purposes?

Reply:
yes this is true. It should never be initialized in the testing setup